### PR TITLE
Update log messages during STI building

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by nak3** [comment]
Please see:

~~~
I0805 08:38:53.623215       1 sti.go:112] Building 172.30.83.253:5000/sinatra/cakephp-example
I0805 08:38:53.624150       1 clone.go:26] Cloning into /tmp/sti581037311/upload/src
I0805 08:39:00.994809       1 docker.go:180] Image registry.access.redhat.com/openshift3/php-55-rhel7:latest available locally
I0805 08:39:00.994833       1 docker.go:267] Image contains io.s2i.scripts-url set to 'image:///usr/local/sti'
I0805 08:39:00.994856       1 download.go:56] Using image internal scripts from: image:///usr/local/sti/assemble
I0805 08:39:00.994864       1 download.go:56] Using image internal scripts from: image:///usr/local/sti/run
I0805 08:39:00.998304       1 docker.go:180] Image registry.access.redhat.com/openshift3/php-55-rhel7:latest available locally
I0805 08:39:00.998324       1 docker.go:267] Image contains io.s2i.scripts-url set to 'image:///usr/local/sti'
I0805 08:39:00.998342       1 download.go:56] Using image internal scripts from: image:///usr/local/sti/save-artifacts
I0805 08:39:00.998353       1 sti.go:182] Using assemble from upload/src/.sti/bin
I0805 08:39:00.998360       1 sti.go:182] Using run from image:///usr/local/sti
I0805 08:39:00.998365       1 sti.go:182] Using save-artifacts from image:///usr/local/sti
I0805 08:39:00.998374       1 sti.go:120] Clean build will be performed
I0805 08:39:00.998378       1 sti.go:123] Performing source build from https://github.com/openshift/cakephp-ex.git
I0805 08:39:00.998382       1 sti.go:130] Building 172.30.83.253:5000/sinatra/cakephp-example
I0805 08:39:00.998389       1 sti.go:326] Using image name registry.access.redhat.com/openshift3/php-55-rhel7:latest
~~~

`Building 172.30.83.253:5000/sinatra/cakephp-example` comes up two times and unclear what it is doing.

File: pkg/build/strategies/sti/sti.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 		}
 	}
 
-	glog.V(1).Infof("Building %s", config.Tag)
+	glog.V(1).Infof("Running STI script in the builder image")
```
Comment (mfojtik) [inline pkg/build/strategies/sti/sti.go]: can you please use the `config.Tag` instead of "the builder image" or just add that information?

**Comment by mfojtik** [comment]
LGTM, one small nit

**Comment by nak3** [comment]
Thank you. I replaced.

File: pkg/build/strategies/sti/sti.go [hunk 2]
```diff
@@ -136,7 +136,7 @@ func (b *STI) Build(config *api.Config) (*api.Result, error) {
 		}
 	}
 
-	glog.V(1).Infof("Building %s", config.Tag)
+	glog.V(1).Infof("Running STI script in %s", config.Tag)
```
Comment (bparees) [inline pkg/build/strategies/sti/sti.go]: please change STI to S2I

**Comment by nak3** [comment]
Thank you again. I fixed it.

**Comment by bparees** [comment]
[merge]

**Pull request merged**